src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -257,7 +257,7 @@
       <Context>ConfiguredBrowseObject</Context>
     </PropertyPageSchema>
 
-    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)ProjectDebugger.xaml">
+    <PropertyPageSchema Include="$(ManagedXamlNeutralResourcesDirectory)ProjectDebugger.xaml">
```

Comment:
>ManagedXamlNeutralResourcesDirectory [](start = 35, length = 36)

If I understand this correctly, the problem in #6331 is that the XlfInput was set to false without updating this Include attribute?

Reply:
Indeed, that and the CommonFiles.swr changes.